components/brave_wallet_ui/page/screens/backup-wallet/intro/backup-wallet-intro.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// utils
+import { getLocale } from '../../../../../common/locale'
+
+// components
+import { Checkbox } from 'brave-ui'
+import NavButton from '../../../../components/extension/buttons/nav-button/index'
+import PasswordInput from '../../../../components/shared/password-input/index'
+
+// hooks
+import { usePasswordAttempts } from '../../../../common/hooks/use-password-attempts'
+
+// style
+import {
+  StyledWrapper,
+  Title,
+  Description,
+  TermsRow,
+  SkipButton,
+  PageIcon,
+  PasswordColumn
+} from './backup-wallet-intro.style'
+import { InputLabelText } from '../../../../components/desktop/popup-modals/account-settings-modal/account-settings-modal.style'
+
+export interface Props {
+  onSubmit: () => void
+  isBackupTermsAccepted: boolean
+  isOnboarding: boolean
+  onSubmitTerms: (key: string, selected: boolean) => void
+  onCancel: () => void
+  recoveryPhraseLength: number
+}
+
+export const BackupWalletIntroStep = ({
+  onSubmit,
+  isBackupTermsAccepted,
+  isOnboarding,
+  onSubmitTerms,
+  onCancel,
+  recoveryPhraseLength
+}: Props) => {
+  // state
+  const [password, setPassword] = React.useState<string>('')
+  const [isCorrectPassword, setIsCorrectPassword] = React.useState<boolean>(true)
+
+  const hasPasswordError = isOnboarding
+    ? false // password not needed during onboarding
+    : !password || !isCorrectPassword // needs correct value
+
+  // custom hooks
+  const { attemptPasswordEntry } = usePasswordAttempts({ maxAttempts: 3 })
```

Comment:
Seems like it may be better to set maxAttempts within this function now that I'm seeing how it's used more. This way we don't end up with different number of max attempts for each time it's called on accident